grazer/src/main/java/edu/asu/diging/grazer/web/PersonController.java

```diff
@@ -75,4 +91,32 @@ public String getPersonGraph(@PathVariable("personId") String personId, Model mo
         model.addAttribute("alternativeIdsString", String.join(",", concept.getAlternativeUris()));
         return "person/graph";
     }
```

Comment:
is this method still needed?

Reply:
This method lists the relationships between the concepts, so I'm not sure if it should be removed